core/domain/rights_manager.py

```diff
@@ -214,11 +213,11 @@ def update_activity_first_published_msec(
     activity_rights.first_published_msec = first_published_msec
     commit_cmds = [{
         'cmd': CMD_UPDATE_FIRST_PUBLISHED_MSEC,
-        'first_published_msec': first_published_msec
+        'first_published': first_published_msec
```

Comment:
Actually I looked at the file. I think the convention, insofar as we have one, is to use old_first_published_msec and new_first_published_msec.